tests/jerry/regression-test-issue-122.js

```diff
@@ -15,10 +15,11 @@
 
 try
 {
-  print(this, []);
+  print({toString: function() { throw new TypeError("foo"); }}, []);
```

Comment:
I see.

Maybe, now we could use the initial version of regression test case from #122:
`print(this, []);`